src/pool/cli.c

```diff
@@ -158,6 +158,7 @@ dc_pool_alloc(unsigned int nr)
 		goto failed;
 	}
 
+	pool->dp_map_version_known = 1;
```

Comment:
just confirm, the initial version should be set as 0 or 1? (seems initially is zero, and every change will +1)

Reply:
I believe 1 is correct. All pool maps begin from version 1. See init_pool_metadata. So every client knows the pool map version must at least be 1. I'll add a comment here.